### PR TITLE
Add support for SELinux AVC statistics

### PR DESCRIPTION
**Pull request by cgzones**
Add interfaces to read the SELinux AVC runtime statistics from /sys/fs/selinux/avc/cache_stats and /sys/fs/selinux/avc/hash_stats.

Refactored from https://github.com/prometheus/node_exporter/pull/2418

/cc @jpds

### CODE REVIEW

File: selinuxfs/avc_cache_stats.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type AVCStat struct {
```
Comment (SuperQ) [inline selinuxfs/avc_cache_stats.go:27]: Would you please add documentation comments to exported structs and functions?

File: selinuxfs/avc_cache_stats.go [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	Lookups     uint64
+	Hits        uint64
+	Misses      uint64
+	Allocations uint64
+	Reclaims    uint64
+	Frees       uint64
+}
+
+func (fs FS) parseAVCStats() (AVCStat, error) {
```
Comment (SuperQ) [inline selinuxfs/avc_cache_stats.go:36]: This should probably be an exported function. I don't see any other way to get an `AVCStat` struct returned.

**Review by SuperQ — CHANGES_REQUESTED** [review]

File: selinuxfs/avc_cache_stats.go (hunk 1 above)
Comment (cgzones) [reply selinuxfs/avc_cache_stats.go:28]: added

File: selinuxfs/avc_cache_stats.go (hunk 2 above)
Comment (cgzones) [reply selinuxfs/avc_cache_stats.go]: true; done

**Review by SuperQ — APPROVED** [review]
LGTM

**Review by discordianfish — APPROVED** [review]

File: selinuxfs/avc_cache_stats.go [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// SELinux access vector cache statistics
```
Comment (SuperQ) [inline selinuxfs/avc_cache_stats.go:27]:
```suggestion
// SELinux access vector cache statistics.
```

**Review by SuperQ — CHANGES_REQUESTED** [review]

**Review by SuperQ — APPROVED** [review]

**Pull request merged**